packages/itwinui-react/src/core/Select/Select.test.tsx

```diff
@@ -514,3 +514,41 @@ it('should use custom render for selected item (multiple)', async () => {
   expect((selectedValues[0] as HTMLElement).style.color).toEqual('green');
   expect((selectedValues[1] as HTMLElement).style.color).toEqual('red');
 });
+
+it('should update live region when selection changes', async () => {
+  const MultiSelectTest = () => {
+    const [selected, setSelected] = React.useState([0]);
+    return (
+      <Select
+        options={[0, 1, 2].map((value) => ({
+          value,
+          label: `Item ${value}`,
+        }))}
+        popoverProps={{ visible: true }}
+        multiple
+        value={selected}
+        onChange={(value, type) =>
+          setSelected((prev) =>
+            type === 'added'
+              ? [...prev, value]
+              : prev.filter((v) => v !== value),
+          )
+        }
+      />
+    );
+  };
+  const { container } = render(<MultiSelectTest />);
```

Comment:
It was way harder than it should have been to write this test. Our Select doesn't maintain its own state - we should change that in the future

Reply:
I think we have an issue to update tests.

Reply:
it's not a problem with tests though - it's the component that needs to be improved